web/server.js

```diff
@@ -45,6 +45,20 @@ module.exports = function(options) {
     });
   });
 
+  function skipCSRF(request, reply) {
+    return reply(true);
```

Comment:
Yeah, looking at the code, this needs to be fixed up: https://github.com/hapijs/crumb/blob/master/lib/index.js#L62